ssl_cert_tracker/db_helper.py

```diff
@@ -13,10 +13,9 @@
 :contact:    ali.rahmat@phsa.ca
 
 """
-
-from datetime import datetime
-import time
 import logging
+from django.utils.dateparse import parse_datetime

```

Comment:
line 49

Fixed

Reply:
so where are the stories dealing with these problems? please hang the stories on the trello board and paste the links here. otherwise (as the romans used to say) "verba volant..."